### PR TITLE
feat: add test scenario for user with no spaces invitations - EXO-66953

### PR DESCRIPTION
**Pull request by ahamdi**

### CODE REVIEW

File: src/test/java/io/meeds/qa/ui/steps/definition/HomeStepDefinition.java [hunk 1]
```diff
@@ -352,6 +352,12 @@ public void isNoConnectionsBadge() {
                                                 .isTrue();
   }
 
+  @Then("^The spaces badge isn't displayed$")
+  public void thereIsNoSpacesBadge() {
+    assertThat(homeSteps.thereIsNoSpacesBadge()).as("The spaces badge shouldn't be displayed")
```
Comment (rdenarie) [inline src/test/java/io/meeds/qa/ui/steps/definition/HomeStepDefinition.java]: after reflexion, assertThat instruction are more in Homepage.java, for example  checkNumberOfConnectionsInDrawer

So can you move the assertThat instruction at this level ? Can you also modify to isNoConnectionBadge ?

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: src/test/java/io/meeds/qa/ui/steps/definition/HomeStepDefinition.java (hunk 1 above)
Comment (ahamdi) [reply src/test/java/io/meeds/qa/ui/steps/definition/HomeStepDefinition.java]: Done

File: src/main/java/io/meeds/qa/ui/pages/HomePage.java [hunk 2]
```diff
@@ -268,8 +268,14 @@ public boolean isPortalDisplayed() {
     return getSiteBody().isCurrentlyVisible();
   }
 
-  public boolean isNoConnectionsBadge() {
-    return getConnectionsBadge().isNotVisible();
+  public void isNoConnectionsBadge() {
+    assertThat(getConnectionsBadge().isNotVisible()).as("The badge shouldn't be displayed")
+            .isTrue();
```
Comment (rdenarie) [inline src/main/java/io/meeds/qa/ui/pages/HomePage.java]:
```suggestion
    assertTrue(getConnectionsBadge().isNotVisible());
```

File: src/main/java/io/meeds/qa/ui/pages/HomePage.java [hunk 3]
```diff
@@ -268,8 +268,14 @@ public boolean isPortalDisplayed() {
...
+  }
+
+  public void thereIsNoSpacesBadge() {
+    assertThat(spacesBadgeElement().isNotVisible()).as("The spaces badge shouldn't be displayed")
+            .isTrue();
```
Comment (rdenarie) [inline src/main/java/io/meeds/qa/ui/pages/HomePage.java]:
```suggestion
    assertTrue(spacesBadgeElement().isNotVisible())
```

File: src/main/java/io/meeds/qa/ui/pages/HomePage.java [hunk 4]
```diff
@@ -268,8 +268,14 @@ public boolean isPortalDisplayed() {
...
   }
 
-  public boolean isNoConnectionsBadge() {
-    return getConnectionsBadge().isNotVisible();
+  public void isNoConnectionsBadge() {
+    assertThat(getConnectionsBadge().isNotVisible()).as("The badge shouldn't be displayed")
+            .isTrue();
+  }
+
+  public void thereIsNoSpacesBadge() {
```
Comment (rdenarie) [inline src/main/java/io/meeds/qa/ui/pages/HomePage.java]:
```suggestion
  public void checkNoSpacesBadge() {
```

File: src/main/java/io/meeds/qa/ui/pages/HomePage.java [hunk 5]
```diff
@@ -268,8 +268,14 @@ public boolean isPortalDisplayed() {
     return getSiteBody().isCurrentlyVisible();
   }
 
-  public boolean isNoConnectionsBadge() {
-    return getConnectionsBadge().isNotVisible();
+  public void isNoConnectionsBadge() {
```
Comment (rdenarie) [inline src/main/java/io/meeds/qa/ui/pages/HomePage.java]:
```suggestion
  public void checkNoConnectionsBadge() {
```

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: src/main/java/io/meeds/qa/ui/pages/HomePage.java [hunk 6]
```diff
@@ -268,8 +268,12 @@ public boolean isPortalDisplayed() {
...
   }
 
-  public boolean isNoConnectionsBadge() {
-    return getConnectionsBadge().isNotVisible();
+  public void checkNoConnectionsBadge() {
+    assertTrue(getConnectionsBadge().isNotVisible());
+  }
+
+  public void checkNoSpacesBadge() {
+    assertTrue(spacesBadgeElement().isNotVisible())
```
Comment (rdenarie) [inline src/main/java/io/meeds/qa/ui/pages/HomePage.java]:
```suggestion
    assertTrue(spacesBadgeElement().isNotVisible());
```

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: src/test/resources/features/Social/SpacesWidget.feature [hunk 7]
```diff
@@ -5,6 +5,15 @@ Feature: Spaces widget checking
   I want to check the spaces in home page
   In order to validate the page
 
+  @smoke
+  Scenario: US 3.3.2_(01) [BACK] No Spaces requests to join
+    Given I am authenticated as 'admin' random user
+    And I create the tenth random user if not existing
```
Comment (rdenarie) [inline src/test/resources/features/Social/SpacesWidget.feature]: the tenth user does not exist in TestInitHook, can you choose an existing one ? 
    String[] randomUsers = new String[] {
        "first",
        "second",
        "third",
        "fourth",
        "fifth",
        "sixth",

**Review by rdenarie — CHANGES_REQUESTED** [review]

**Review by rdenarie — APPROVED** [review]

**Pull request merged**